iron-component-page.html

```diff
@@ -317,7 +317,7 @@
            */
           transitive: {
             type: Boolean,
-            value: false
+            value: true
```

Comment:
The component-page still works more than fast enough for normal use.

This property is needed to make local docs complete and accurate. At present, any element that depends on a behavior from another repo won't get that behavior's information displayed when viewing the component page locally.

The catalog doesn't need this property since it doesn't invoke the analyzer.

An option that works without breaking the API is to use tedium to explicitly set this property to "true" on our repos.